src/Facebook/FacebookRedirectLoginHelper.php

```diff
@@ -100,6 +100,10 @@ public function getLoginUrl($scope = array(), $version = null, $displayAsPopup =
       'sdk' => 'php-sdk-' . FacebookRequest::VERSION,
       'scope' => implode(',', $scope)
     );
+
+    if (!empty($reauthenticate)) {
```

Comment:
Considering documentation, "auth_type" parameter only accept two values which are "reauthenticate" and "https". Rather than passing "null" or "false" and having a result I can't predict, I added the first "if" condition.

The second one allow to use "true" to set "reauthenticate", or passing one of the two expected values ("reauthenticate" or "https").